tests/hazmat/primitives/test_dh.py

```diff
@@ -18,6 +19,11 @@
 from ...utils import load_nist_vectors, load_vectors_from_file
 
 
+skip_no_dhx = pytest.mark.skipif(
```

Comment:
I don't have OpenSSL 1.0.1 on my machine so I will try to set up an environment with OpenSSL 1.0.1 and check that.